code/e2e-tests/json-files.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+import process from 'process';
+
+const storybookUrl = process.env.STORYBOOK_URL || 'http://localhost:8001';
+
+test.describe('JSON files', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(storybookUrl);
+  });
+
+  test('should have index.json', async ({ page }) => {
```

Comment:
would you mind adding one extra case for `stories.json`? Should behave just the same

Reply:
if that is required, let's do that in another PR 👍